website/src/components/canvas/sidebar/ResizeTextfields.jsx

```diff
@@ -104,37 +143,43 @@ class ResizeTextfields extends React.Component {
                   iconStyle={styles.smallIcon}
                   style={styles.small}
                   onTouchTap={this.generateButtonHandler(-1 * ONE_UNIT, HEIGHT)}
+                  disabled={this.state.disableTextfields}
                 >
                     <RemoveCircle/>
                 </IconButton>
                 <TextField
                   onChange={this.handlerTextfieldHeight}
                   floatingLabelText="Height"
                   style={styles.field}
+                  disabled={this.state.disableTextfields}
                 />
                 <IconButton
                   iconStyle={styles.smallIcon}
                   style={styles.small}
                   onTouchTap={this.generateButtonHandler(ONE_UNIT, HEIGHT)}
+                  disabled={this.state.disableTextfields}
```

Comment:
Given that this is the default behavior as per devs and designers of the React Material-UI, I think this should be the default behavior. Everything they implement is based on the Material Design standards.